systemtest/src/test/java/io/strimzi/systemtest/AllNamespaceST.java

```diff
@@ -72,6 +72,29 @@ void testDeployMirrorMakerAcrossMultipleNamespace() {
         checkMirrorMakerForKafkaInDifNamespaceThanCO();
     }
 
+    @Test
+    @Tag(REGRESSION)
+    void testDeployKafkaConnectInOtherNamespaceThanCO() {
+        // Deploy Kafka in other namespace than CO
+        secondNamespaceResources.kafkaEphemeral(CLUSTER_NAME, 3).done();
+        // Deploy Kafka Connect in other namespace than CO
+        secondNamespaceResources.kafkaConnect(CLUSTER_NAME, 1).done();
+        // Check that Kafka Connect was deployed
+        kubeClient.waitForDeployment(kafkaConnectName(CLUSTER_NAME), 1);
+    }
+
+    @Test
+    @Tag(REGRESSION)
+    void testUOWatchingOtherNamespace() {
+        LOGGER.info("Creating user in other namespace than CO and Kafka cluster with UO");
+        secondNamespaceResources.tlsUser(CLUSTER_NAME, USER_NAME).done();
+
+        String previousNamespace = kubeClient.namespace(SECOND_NAMESPACE);
+        kubeClient.waitForResourceCreation("KafkaUser", USER_NAME);
```

Comment:
I'm not sure this really tests the User Operator - if I read it correctly, you create the `KafkaUser` resource and check it exists. Of course it should exist if it was just created. You should instead wait for the `Secret` which the UO creates. Or does the `waitForResourceCreation` method do that?

Reply:
Yeah, this is a good point. We need to check the `Secret` creation for a new user, and it will be a good assert for UO. 
Added this assert, so I can resolve this conversation,